extensions/amp-date-picker/validator-amp-date-picker.protoascii

```diff
@@ -0,0 +1,95 @@
+#
+# Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-date-picker
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-date-picker"
+    allowed_versions: "0.1"
+    allowed_versions: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-date-picker>
+  tag_name: "AMP-DATE-PICKER"
+  requires_extension: "amp-date-picker"
+  attrs: {
+    name: "mode"
+    value_regex: "overlay|static|"
+  }
+  attrs: {
+    name: "type"
+    value_regex: "range|single|"
```

Comment:
Is there a better way to specify "value1, value2 or empty"?

Reply:
Unfortunately no. Please also consider using `casei` for case insensitive matching.